projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -47,7 +47,12 @@ export default function PublicizeForm( {
 
 	const Wrapper = isPublicizeDisabledBySitePlan ? Disabled : Fragment;
 
-	const brokenConnections = connections.filter( connection => false === connection.is_healthy );
+	const brokenConnections = connections.filter(
+		connection => false === connection.is_healthy && 'broken' === connection.error_code
```

Comment:
Is is_healthy ever not a boolean? We could change these checks to `! connection.is_healthy`

Reply:
Yeah I think it was changed, I remember I added this for a reason back then, but probably it's not needed anymore